functional/keylime-non-default-ports/test.sh

```diff
@@ -110,6 +110,8 @@ _EOF"
         rlRun "limeStopAgent"
         rlRun "limeStopRegistrar"
         rlRun "limeStopVerifier"
+        #remove keylime_port_t label from non default ports
+        rlRun "semanage port -D"
```

Comment:
Can we remove just labels from ports we have configured? I am afraid that doing this could possible collide with any a different system setup we might do.

Reply:
Sure, I changed that..